src/main/java/uk/gov/ch/developer/docs/controller/BaseController.java

```diff
@@ -11,6 +11,7 @@
     protected static final Logger LOGGER = LoggerFactory
             .getLogger(BaseController.class.getName());
 
+    //ToDo: Can this be removed?
     protected static final String ERROR_VIEW = "error";
 
```

Comment:
Does removing it break the code?

Reply:
That was more a question "Why was this added in the first place because its not used."